hugegraph-core/src/main/java/com/baidu/hugegraph/task/HugeTaskScheduler.java

```diff
@@ -211,6 +211,12 @@ public boolean close() {
     }
 
     public <V> HugeTask<V> deleteTask(Id id) {
+        HugeTask<?> task = this.tasks.get(id);
+        if (task != null) {
+            E.checkState(task.completed(),
+                         "Can't delete task '%s' in status %s",
+                         id, task.status());
```

Comment:
call remove(Id) to delete task in memory

Reply:
task in memory would be deleted when task is completed